mlflow/ml-package-versions.yml

```diff
@@ -168,6 +169,9 @@ xgboost:
     requirements:
       ">= 0.0.0": ["scikit-learn", "matplotlib"]
       "< 1.6": ["pandas<2"]
+      "== dev": ["setuptools==59.8.0"]
```

Comment:
this is for addressing a setuptools bug: https://github.com/mlflow/mlflow/actions/runs/10173505108/job/28137876205?pr=12832#step:12:274

Reply:
this issue is reported here: https://github.com/scikit-learn/scikit-learn/issues/26992

can you try importing `setuptools` first before `mlflow.sklearn.autolog`?

Reply:
Tested, this way " importing setuptools first before mlflow.sklearn.autolog" does not address it: https://github.com/mlflow/mlflow/actions/runs/10174566035/job/28140563273#step:12:132

Reply:
@WeichenXu123 thanks! can you reproduce this on your machine?

Reply:
No. I can't repro it locally.